docs/building/MlNetMklDeps/version.txt

```diff
@@ -0,0 +1,6 @@
+mac: 2018.1.126
```

Comment:
I would also start putting the MlNetMklDeps version in here too, so we can map it. For example:

```
MlNetMklDeps v0.0.0.4-0.0.0.6
-----
mac: 2018.1.126
linux: 2018.3.222
windows: 2018.3.210
```